### PR TITLE
use julia language support for travis so tests start faster

### PR DESCRIPTION
**Pull request by sjkelly**

### CODE REVIEW

**Comment by juliohm** [comment]
I think this is a feature! We don't want implicit conversions with mixed
coordinates. So it is a good method to guarantee performance. I'll review
it tomorrow carefully. Thanks for awesome contribution!

File: test/points.jl [hunk 1]
```diff
@@ -1,51 +1,86 @@
...
   @test embeddim(Point([1,2])) == 2
   @test coordtype(Point([1,2])) == Int
   @test coordtype(Point([1.,2.])) == Float64
 
   @test embeddim(Point([1,2,3])) == 3
   @test coordtype(Point([1,2,3])) == Int
   @test coordtype(Point([1.,2.,3.])) == Float64
+  
+  # check all 1D Point constructors, because those tend to make trouble
+  Point(1) == Point((1,)) == Point([1])
```
Comment (FPGro) [inline test/points.jl]: Oh it seems like I forget the @test before the tests here

File: src/points.jl [hunk 2]
```diff
@@ -17,28 +17,35 @@ O = Point(0.0, 0.0) # origin of 2D Euclidean space
...
 """
 struct Point{Dim,T}
   coords::SVector{Dim,T}
+  Point{Dim,T}(coords::AbstractVector) where {Dim,T} = new{Dim,T}(coords)
 end
 
 # convenience constructors
 Point{Dim,T}(coords::NTuple{Dim,V}) where {Dim,T,V} = Point{Dim,T}(SVector(coords))
-Point{Dim,T}(coords::Vararg{V,Dim}) where {Dim,T,V} = Point{Dim,T}(SVector(coords))
+Point{Dim,T}(coords...) where {Dim,T} = Point{Dim,T}(coords)
```
[the remark this answers is not in the document]
Comment (FPGro) [reply src/points.jl]: `coords::Vararg{V,Dim}` is too specific again, so it gets ambigous with the inner constructor over the array: 
```julia
ERROR: LoadError: MethodError: Point{1, Int64}(::SVector{1, Int64}) is ambiguous. Candidates:
  Point{Dim, T}(coords::AbstractVector{T} where T) where {Dim, T} in Main at C:\Users\gross\Desktop\meshes.jl:5
  Point{Dim, T}(coords::Vararg{V, Dim}) where {Dim, T, V} in Main at C:\Users\gross\Desktop\meshes.jl:15
Possible fix, define
  Point{1, T}(::V) where {T, V<:(AbstractVector{T} where T)}
Stacktrace:
 [1] Point(coords::Tuple{Int64})
   @ Main ~\Desktop\meshes.jl:10
...
```

[the remark this answers is not in the document]
Comment (FPGro) [reply src/points.jl]: Okay dropping only `Dim` but keeping `::Vararg{V}` seems to work. I'll commit that and see if tests pass.

**Pull request merged**